client/core/notification.go

```diff
@@ -269,6 +269,8 @@ const (
 	TopicFailedCancel         Topic = "FailedCancel"
 	TopicOrderLoaded          Topic = "OrderLoaded"
 	TopicOrderRetired         Topic = "OrderRetired"
+	TopicOrderError           Topic = "OrderError"
```

Comment:
I think this should be more specific.  `TopicQueueOrderFailure` or `TopicFailedOrderSubmission`, etc.
In the future something as generic sounding as `OrderError` will be misused.